pyne/ensdf.py

```diff
@@ -68,6 +67,106 @@ def _to_id(nuc):
         return 0
     return nucid
 
+time_conv_dict = {'as': 1e-18,
```

Comment:
This can probably be removed and then imported from utils.  No reason  to have the data structure twice :)